mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java

```diff
@@ -104,4 +122,73 @@ protected void initializeMailboxManager() throws Exception {
         storeMailboxManager.setMessageSearchIndex(messageSearchIndex);
         storeMailboxManager.init();
     }
+
```

Comment:
If you like organic chemistry name, here is one of the longest one: https://fr.wikipedia.org/wiki/Nom_le_plus_long and here you find a guy speeding 3h30 saying it https://www.youtube.com/watch?v=IHVWRZQWdD8